clients/js/spec/zawgyi_converter_spec.js

```diff
@@ -546,7 +546,7 @@ var zawygi_unicode_convert_data = [
       u: "ဝေသစ်လွင် about"
     },
     {  // #101 From case #134: Other
-      z: "ျပန္လမ္းမဲ့ကၽြန္းအပိုင္း၄",
+      z: "ျပန္လမ္းမဲ့ကြၽန္းအပိုင္း၄",
```

Comment:
Again, why change the Z test string? Also, please add the same test strings as in the Java test file.

Reply:
Same case as in Java. I will add the same test strings as in the Java test file.